Tests/js/models/assets/ez-locationmodel-tests.js

```diff
@@ -1123,6 +1123,45 @@ YUI.add('ez-locationmodel-tests', function (Y) {
         },
     });
 
+    toObjectTest = new Y.Test.Case({
+        name: "eZ location Model toObject test",
+
+        setUp: function () {
+            this.contentInfoObject = {id: 42, name: 'aspirine'};
+            this.contentInfo = new Y.Mock({Content: {alwaysAvailable: false}});
+            Mock.expect(this.contentInfo, {
+                method: 'toObject',
+                returns: this.contentInfoObject
+            });
+
+            this.location = new Y.eZ.Location({
+                locationId: 42,
+                contentInfo: this.contentInfo
+            });
+        },
+
+        tearDown: function () {
+            this.location.destroy();
+        },
+
+        "Should return an object based on the content": function () {
+            var object = this.location.toObject();
+
+            Assert.areEqual(
+                object.id, this.location.get('locationId'),
+                "The object should have an id"
+            );
+            Assert.areSame(
```

Comment:
you can replace those 2 assertions by one comparing `object.contentInfo` and `this.contentInfoObject`